opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/FunctionParameterRepository.java

```diff
@@ -166,6 +167,14 @@ public class FunctionParameterRepository {
           .put("quote_field_suffix", (b, v) -> b.quoteFieldSuffix(v.stringValue()))
           .build();
 
+  public static final Map<String, RelevanceQuery.QueryBuilderStep<WildcardQueryBuilder>>
+      WildcardQueryBuildActions = ImmutableMap.<String,
+          RelevanceQuery.QueryBuilderStep<WildcardQueryBuilder>>builder()
+      .put("boost", (b, v) -> b.boost(convertFloatValue(v, "boost")))
+      .put("case_insensitive", (b, v) -> b.caseInsensitive(convertBoolValue(v, "case_insensitive")))
+      .put("rewrite", (b, v) -> b.rewrite(checkRewrite(v, "rewrite")))
```

Comment:
Should the accepted strings for rewrite be case insensitive?

Reply:
No, it could be even cAMeLcASe.